pkg/cmd/server/admin/overwrite_bootstrappolicy.go

```diff
@@ -0,0 +1,153 @@
+package admin
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/meta"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/tools"
+	"github.com/openshift/origin/pkg/api/latest"
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	authorizationetcd "github.com/openshift/origin/pkg/authorization/registry/etcd"
+	roleregistry "github.com/openshift/origin/pkg/authorization/registry/role"
+	rolebindingregistry "github.com/openshift/origin/pkg/authorization/registry/rolebinding"
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	configapilatest "github.com/openshift/origin/pkg/cmd/server/api/latest"
+	"github.com/openshift/origin/pkg/cmd/server/etcd"
+	cmdclientcmd "github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	templateapi "github.com/openshift/origin/pkg/template/api"
+)
+
+type OverwriteBootstrapPolicyOptions struct {
+	File             string
+	MasterConfigFile string
+}
+
+func NewCommandOverwriteBootstrapPolicy() *cobra.Command {
+	options := &OverwriteBootstrapPolicyOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "overwrite-policy",
+		Short: "Overwrite policy for OpenShift.  DANGER: THIS BYPASSES ALL ACCESS CONTROL CHECKS AND WRITES DIRECTLY TO ETCD!",
+		Run: func(c *cobra.Command, args []string) {
+			if err := options.Validate(args); err != nil {
+				fmt.Println(err.Error())
+				c.Help()
+				return
+			}
+
+			if err := options.OverwriteBootstrapPolicy(); err != nil {
+				glog.Fatal(err)
+			}
+		},
+	}
+
+	flags := cmd.Flags()
+
+	flags.StringVar(&options.File, "filename", DefaultPolicyFile, "The policy template file containing roles and bindings.")
```

Comment:
default these or make them specify them? I'm hesitant to default anything for this command